### ISSUE TITLE
Blur passes not respected for layers

### ISSUE THREAD

**Issue by MonstrousOgre** [hyprwm/Hyprland#4561]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch HEAD at commit 03ebbe18ed8517ee22591eac82cd54322f42cb7d  (props: bump ver to 0.34.0).
Date: Mon Jan 1 12:03:15 2024
Tag: v0.34.0

flags: (if any)

```

</details>

### Bug or Regression?

Bug

### Description

Here is my relevant blur config

```
    blur {
        enabled = yes
        size = 8
        passes = 2
        new_optimizations = on
        popups = true
    }
```
I also have `blurls=osd` in my config. When blurring the osd layer (or any layer really), it only uses 1 pass. It uses 2 passes for windows though.

### How to reproduce

1. Set blur passes to anything greater than 1
2. Blur a layer
3. Notice that the blur looks weird

### Crash reports, logs, images, videos

Here is a region of the wallpaper blurred by a window:

![image](https://github.com/hyprwm/Hyprland/assets/16362557/ca10a9cc-c185-4814-8af2-7110c42112f7)

This is the same region blurred by a layer:

![image](https://github.com/hyprwm/Hyprland/assets/16362557/4083eca8-a19d-4264-accd-9f32a87cbb2e)

**Comment by RyanOrigens** [comment]
[MonstrousOgre](https://github.com/MonstrousOgre), What are you using for making this Pop up Menu? Sorry i know it isn't related to the issue, but i'm kinda curious...

**Comment by tchofy** [comment]
I'd hazard a guess that's https://github.com/Aylur/ags, since they have it's folder on their dotfiles repo. Another guess would be eww, but since they don't have an eww folder, it's most likely ags.

**Comment by MonstrousOgre** [comment]
Yeah it's ags. I'm using it for a bar, an OSD, and a notification popup. It's pretty neat.
